factorial.c

```diff
@@ -1,14 +1,18 @@
+/*code to find factorial of a number*/
+
+
+
 #include <stdio.h>
 #include <stdlib.h>
 
 int main()
 {
-    int num,i,fact=1;
+    int num,i,fact=1;/*declare them as integer type*/
     printf("enter the num: ");
     scanf("%d",&num);
-    for(i=1;i<=num;i++)
+    for(i=1;i<=num;i++)/*run the loop from 1 to the number we entered,so that it keeps on muliplying*/
```

Comment:
can we start with 2 instead of 1 here ?

Reply:
yeah we can start with 2, any number multiplied by one is the number itself. so if we start with 2 it wont make any change in the final answer.